site/src/pages/components/radio.research.mdx

```diff
@@ -38,35 +39,35 @@ After the research there are still [discussions to be made](#discussion), where
 
 ## Anatomy
 
-<Anatomy component="Radio Button" />
+<Anatomy component="Radio" />
 
 ## Concepts
 
-<Concepts client:load component="Radio Button" />
+<Concepts client:load component="Radio" />
 
 ## Discussion
 
-At a point during the research, the scope of this study os shifted from the “radio button” to the “radio button group”, because there is no particular strong use case for a standalone “radio button”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio button group”, being composed of two or more radio buttons.
+At a point during the research, the scope of this study os shifted from the “radio” to the “radio group”, because there is no particular strong use case for a standalone “radio”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio group”, being composed of two or more radios.
 
 Also based on this fact and on the current state of the design systems, a more final anatomy was proposed.
 
 ### Proposed Anatomy - by Tag Hierarchy
 
-This section shows the proposed visual constituents for the _Radio Button Group_, which is here considered to be the actual component, while the radio buttons are a part of the group.
+This section shows the proposed visual constituents for the _Radio Group_, which is here considered to be the actual component, while the radios are a part of the group.
 
-- Radio Button Group (the actual component)
+- Radio Group (the actual component)
   - Group Label (functions as a label for the whole group)
   - Required Indicator
-  - Radio Button (at least two required)
+  - Radio (at least two required)
```

Comment:
while I like this break down, we should have some text that explains why two are required.